src/lib/sql/sql_translator.cpp

```diff
@@ -132,23 +132,14 @@ bool is_trivial_join_predicate(const AbstractExpression& expression, const Abstr
 
 namespace opossum {
 
-SQLTranslator::SQLTranslator(const UseMvcc use_mvcc)
+SQLTranslator::SQLTranslator(const UseMvcc use_mvcc,
+                             const std::shared_ptr<std::unordered_map<std::string, std::shared_ptr<Table>>> meta_tables)
     : SQLTranslator(use_mvcc, nullptr, std::make_shared<ParameterIDAllocator>(),
-                    std::unordered_map<std::string, std::shared_ptr<LQPView>>{}) {}
+                    std::unordered_map<std::string, std::shared_ptr<LQPView>>{}, meta_tables) {}
 
-std::vector<ParameterID> SQLTranslator::parameter_ids_of_value_placeholders() const {
-  const auto& parameter_ids_of_value_placeholders = _parameter_id_allocator->value_placeholders();
-  auto parameter_ids = std::vector<ParameterID>{parameter_ids_of_value_placeholders.size()};
-
-  for (const auto& [value_placeholder_id, parameter_id] : parameter_ids_of_value_placeholders) {
-    parameter_ids[value_placeholder_id] = parameter_id;
-  }
-
-  return parameter_ids;
-}
+SQLTranslationResult SQLTranslator::translate_parser_result(const hsql::SQLParserResult& result) {
+  _cacheable = true;
```

Comment:
Do we need that since `_cacheable` default to `true`?

Reply:
imo yes. since one parser can be used to translate multiple queries subesquently, the cacheable attribute needs to be reset at each call. instead we can omit the initialization

Reply:
Sounds reasonable, it should stay as is.